Mathlib/Algebra/Category/ModuleCat/Presheaf.lean

```diff
@@ -177,6 +203,11 @@ instance : Preadditive (PresheafOfModules R) where
 
 end Hom
 
+lemma naturality_apply {M M' : PresheafOfModules R} (f : M ⟶ M')
+    {X Y : Cᵒᵖ} (g : X ⟶ Y) (x : M.obj X) :
```

Comment:
To be consistent with the naming in the rest of this file, `M` and `M'` could be called `P` and `Q` here. And should this also be in the `Hom` namespace? Then dot notation works on `f`?

Reply:
I have changed `M` and `M'` to `P`, `Q`, but I am not so sure about the namespace change because dot does not work in that case (Lean does not see that ` P ⟶ Q` is also `Hom P Q`).